roles/flannel_register/defaults/main.yaml

```diff
@@ -0,0 +1,11 @@
+---
+flannel_network: "{{ openshift.master.portal_net | default('172.16.1.1/16', true) }}"
+flannel_min_network: "{{ min_network | default('172.16.5.0', true) }}"
+flannel_subnet_len: "{{ subnet_len | default(24, true) }}"
+flannel_etcd_key: /openshift.com/network
+etcd_hosts: "{{ etcd_urls }}"
+etcd_conf_dir: "{{ openshift.common.config_base }}/master"
+etcd_peer_ca_file: "{{ etcd_conf_dir }}/ca.crt"
```

Comment:
This will be `"{{ etcd_conf_dir }}/master.etcd-ca.crt"` when not using embedded etcd - something like...

```
"{{ etcd_conf_dir + '/ca.crt' if (openshift.master.embedded_etcd | bool) else etcd_conf_dir + '/master.etcd-ca.crt' }}"
```